tools/RawEchoServer.re

```diff
@@ -59,11 +59,17 @@ let request_handler:
       Httpaf.Reqd.respond_with_string(reqd, res, content);
     };
     open Httpkit.Server.Middleware;
-    let ctx = {closer, req, respond, state: App.initial_state};
+    let ctx = {
+      closer,
+      req,
+      respond,
+      body: () => None,
```

Comment:
I think that to have feature parity, this should be calling `Httpkit_lwt.Server.Request.read_body(req)` like we do in `lwt/server/Http.re`.

So this would be something like

```reason
Httpkit_lwt.Server.Request.read_body(req)
>|= ( body_string => {
  /* use    () => body_string    here instead of    () => None */
})
|> ignore;
```

Reply:
Ah right, I left it as a stub and forgot to update it. Will do it now.

Btw, what exactly is the purpose of that example? Just to show how to build e.g. the request_handler on your own?

Reply:
Benchmarking primarily, secondly to show how you'd write stuff without httpkit.

Reply:
Alright, fixed